lib/turn.rb

```diff
@@ -6,6 +6,24 @@ def initialize(player, board)
     @board = board
   end
 
+  def get_move
+    move = gets.chomp.downcase
+    columns = ["a", "b", "c", "d", "e", "f", "g"]
+    while !columns.include?(move)
+      puts "Oops, that's an invalid move. Please select column A-G!"
+      move = gets.chomp.downcase
+    end
+    @board.board_grid.reverse.each do |row|
```

Comment:
ok this is more of a clarifying comment for me to write this out loud, but we're reversing because we want the first move in a column to do into the lowest row I.e. [5][0]. Correct?